### ISSUE TITLE
Local daemon crashes when sending transaction with mixin over 0

### ISSUE THREAD

**Issue by frankiexile** [cryptonotefoundation/cryptonote#264]
Assertion failed: ofa.outs.size() && "internal error: ofa.outs.size() is empty", file C:\Paradigm\src\Rpc\RpcServer.cpp, line 270

**Comment by frankiexile** [comment]
does anyone have a solution for this? im just trying to test transactions and i can't figure out where this error is coming from

**Comment by frankiexile** [comment]
the only thing i can think of is when i changed the CRYPTONOTE_BLOCK_GRANTED_FULL_REWARD_ZONE to 1000000 to increase the max transaction size because i couldnt send more than 500000 coins at a time. dont know if this has anything to do with it but if it does what is the proper way to increase the max tx size

**Comment by SoraKohaku** [comment]
on max trans size already just make fee balance

**Comment by frankiexile** [comment]
are you saying that i need to make the minimum fee higher?

**Comment by SoraKohaku** [comment]
yep cuz if over  balance of transaction will become orphan block

**Comment by frankiexile** [comment]
ok so CRYPTONOTE_MAX_TX_SIZE                        = 1000000000;

does that mean that  minimum fee needs to be   = 1000000000; as well?

**Comment by frankiexile** [comment]
block granted full reward = 100000 also

**Comment by frankiexile** [comment]
after doing some tests i think this issue was linked to having 2 wallets on the same machine running on the same local daemon. when i sent from wallet 1 to wallet 2 everything was fine, then i sent from wallet 2 back to wallet 1 with a mixin of over 0 and i get this crash error. then it doesnt allow me to send with a mixin over 0 from wallet 1 anywhere either. only until after i deleted wallet 2 off this machine and sent money to a wallet on a completely different computer the issue completely went away as far as i know.

**Comment by SoraKohaku** [comment]
no used for 2 wallet you need 1 wallet 1 machine and many adress accept that rule of decentralize. 1 app 1 options. but heavy and Cn-7 difficulty is great even processor is hardening. just GPU is fast. need clarify again for this~between CPU for old times fast now they priority GPU Heavy, Lite and CN-7 PoW.

try look my repo v3 if interested on CN-7. for heavy is V2. we not sure what we follow for prevent or accept asic who know. just inform...

fee no need

**Comment by SoraKohaku** [comment]
why you need 1 machine 2 wallet. used lite bro with python programs
